### DISCUSSION TITLE
【羊村】组织公告栏

### DISCUSSION THREAD

**Discussion by jiangzhonglian** [apachecn/home#204] [Announcements]
## 组织公告栏

- 加群方式：<https://www.apachecn.org/join>
- 奖励机制：<https://github.com/orgs/apachecn/discussions/243>
- 组织成员榜单：<https://github.com/orgs/apachecn/discussions/244>

> 正在翻译

- Pytorch 2.0 翻译活动：<https://github.com/apachecn/pytorch-doc-zh/discussions/583>

> 招聘求职（可自行添加）

- 面试指南：<https://interview.apachecn.org>
- 公司避雷：<https://www.kdocs.cn/l/ce7Bky0T3phy>
- 招聘(找员工)：<https://github.com/orgs/apachecn/discussions/216>
- 求职(找工作)：<https://github.com/orgs/apachecn/discussions/237>

这个地方用于收集大家的奇思妙想（欢迎留言），一起加油 💪.

**Comment by jiangzhonglian** [comment]
## 网站已经完成升级：

-  更新时间: 2023-11-09

> chatgpt

- <https://sd.apachecn.org>

>  中文系列教程

- ZetCode 中文系列教程: <https://zetcode.apachecn.org>
- GeeksForGeeks 人工智能中文教程: <https://gfg2ai.apachecn.org>
- GeeksForGeeks 编程语言杂项中文教程: <https://gfg2lm.apachecn.org>

> IT金融

- <http://web3.apachecn.org>
- <https://quant.apachecn.org> 
- <https://backtrader.apachecn.org>
- <https://portfolio.apachecn.org>


> 基础语言

- <https://think2python.apachecn.org>
- <https://numba.apachecn.org>
- <https://cython.apachecn.org>
- <https://seaborn.apachecn.org>
- [GeeksForGeeks C# 中文教程📚](https://apachecn.github.io/geeksforgeeks-csharp-zh)
- [GeeksForGeeks Scala 中文教程📚](https://apachecn.github.io/geeksforgeeks-scala-zh)
- [GeeksForGeeks Python 中文教程📚](https://apachecn.github.io/geeksforgeeks-python-zh)
- [GeeksForGeeks C/C++ 中文教程📚](https://apachecn.github.io/geeksforgeeks-c-cpp-zh)
- [GeeksForGeeks Java 中文教程📚](https://apachecn.github.io/geeksforgeeks-java-zh)
- [GeeksForGeeks JavaScript 中文教程📚](https://apachecn.github.io/geeksforgeeks-js-zh)
- [GeeksForGeeks Web 杂项中文教程📚](https://apachecn.github.io/geeksforgeeks-web-misc-zh)
- [GeeksForGeeks Angular/Vue/React 中文教程📚](https://apachecn.github.io/geeksforgeeks-ng-vue-react-zh)
- [GeeksForGeeks jQuery 中文教程📚](https://apachecn.github.io/geeksforgeeks-jquery-zh)
- [GeeksForGeeks CSS 中文教程📚](https://apachecn.github.io/geeksforgeeks-css-zh)
- [GeeksForGeeks HTML 中文教程📚](https://apachecn.github.io/geeksforgeeks-html-zh)
- [GeeksForGeeks ASP 中文教程📚](https://apachecn.github.io/geeksforgeeks-asp-zh)
- [GeeksForGeeks SQL 中文教程📚](https://apachecn.github.io/geeksforgeeks-sql-zh)
- [GeeksForGeeks NodeJS 中文教程📚](https://apachecn.github.io/geeksforgeeks-nodejs-zh)
- [GeeksForGeeks PHP 中文教程📚](https://apachecn.github.io/geeksforgeeks-php-zh)


> 大数据

- <https://storm.apachecn.org>
- <https://flink.apachecn.org>
- <https://zeppelin.apachecn.org>
- <https://airflow.apachecn.org>
- <https://spark.apachecn.org>
- <https://kafka.apachecn.org> | <https://kafka1x.apachecn.org>

> 求职

- <https://biz5min.apachecn.org>
- <https://idw.apachecn.org>
- <https://interview.apachecn.org> | <https://baguwen.apachecn.org>
- <https://rms.apachecn.org> | <https://rms1.apachecn.org> | <https://rms2.apachecn.org> | <https://rms3.apachecn.org> | <https://rms4.apachecn.org>

> 数据科学

- <https://algo.apachecn.org>
- <https://think2ds.apachecn.org>
- <https://linalg.apachecn.org>
- <https://fe4ml.apachecn.org> | <https://pyda.apachecn.org> | <https://hands1ml.apachecn.org>

> 机器学习

- <https://sklearn.apachecn.org>
- <https://gensim.apachecn.org>
- <https://lightgbm.apachecn.org>
- <https://xgboost.apachecn.org>
- <https://pytorch.apachecn.org> | <https://pytorch0x.apachecn.org>
- <https://nlp2pytorch.apachecn.org>
- <https://opencv.apachecn.org> | <https://cv.apachecn.org>

**Reply by jiangzhonglian** [reply]
待升级:  hbase  + scrapy + hudi + gitlab+ fasttext + kudu + elasticsearch + the-way-to-wealth-freedom-notes + web-hacking-101-zh + think-py-2e-zh + geeksforgeeks-lang-misc-zh 文档需要做升级

**Reply by jiangzhonglian** [reply]
https://guru99.apachecn.org

**Reply by jiangzhonglian** [reply]
https://smm.apachecn.org/#/

**Reply by jiangzhonglian** [reply]
更新？

**Comment by jiangzhonglian** [comment]
1

**Reply by lyc666-beep** [reply]
2
